backend/config.json.example

```diff
@@ -118,7 +118,8 @@
       "type": "vault",
       "host": "http://vault1:8200/",
       "prefix": "/v1/",
-      "backend": "secret/"
+      "backend": "secret/",
+      "ttl": 600000
```

Comment:
Changing the config file from a `json` to a `js` file sound reasonable and gives the user better understanding when looking at the configurations.

I would stick with ms for the TTL.